saleor/graphql/page/sorters.py

```diff
@@ -19,6 +20,10 @@ def description(self):
             description = f"Sort pages by {sort_name}."
             if self.name == "PUBLICATION_DATE":
                 description += DEPRECATED_IN_3X_INPUT
+            if self.name == "CREATION_DATE":
+                description += DEPRECATED_IN_3X_INPUT
```

Comment:
Does that make sense to introduce a new filter and instantly deprecate it? Why not only introduce the non-deprecated one?